ansible/roles/base_station_services/tasks/main.yml

```diff
@@ -31,8 +31,11 @@
   systemd: name={{item}} enabled=yes
   with_items:
     - rover-base_station_gui.service
+    - rover-base_station_gstreamer.service
+    - rover-base_station_gstreamer_2.service
     - rover-lcm_bridge_server.service
+    - rover-base_station_microscope.service
```

Comment:
So are we removing microscope or no?

Reply:
yeah remove, did you push the microscope fix to this branch?

Reply:
yes but i think this fix is why camera switching is soooo much slower

Reply:
actually don't delete it quite yet...i actually still think we should have a microscope button since we can uniquely identify it by it's name

Reply:
sure